`npm run build` is currently failing

Outputs:

```
.
.
.
.
.
.
    The expected type comes from property 'about' which is declared here on type 'UpdateAgentRequest'
src/managers/agent.ts:113:7 - error TS2322: Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.

113       name,
          ~~~~

  src/api/models/UpdateAgentRequest.ts:21:3
    21   name: string;
         ~~~~
    The expected type comes from property 'name' which is declared here on type 'UpdateAgentRequest'

src/index.ts

    at getRollupError (/lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup/dist/shared/parseAst.js:282:41)
    at Object.error (/lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup/dist/shared/parseAst.js:278:42)
    at Object.error (/lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup/dist/shared/rollup.js:813:32)
    at RollupContext.error (/lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup-plugin-typescript2/src/context.ts:35:17)
    at /lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup-plugin-typescript2/src/diagnostics.ts:70:17
    at Array.forEach (<anonymous>)
    at printDiagnostics (/lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup-plugin-typescript2/src/diagnostics.ts:42:14)
    at typecheckFile (/lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup-plugin-typescript2/src/index.ts:67:3)
    at Object.<anonymous> (/lab/github.com/julep-ai/infuse/sdks/ts/node_modules/rollup-plugin-typescript2/src/index.ts:269:5)
    at Generator.next (<anonymous>)
```

--- a/.github/workflows/lint-and-format.yml
+++ b/.github/workflows/lint-and-format.yml
@@ -95,7 +95,7 @@ jobs:
       - name: Build TypeScript SDK
         run: |
           cd sdks/ts
-          # npm run build
+          npm run build
 
       - uses: stefanzweifel/git-auto-commit-action@v4
         with:

